ideascube/mediacenter/management/commands/import_medias.py

```diff
@@ -93,21 +93,32 @@ def add(self, metadata):
                                 title)
             return
 
-        path = os.path.join(self.ROOT, original)
-        if not os.path.isfile(path):
-            self.report.error(u'Path not found', path)
-            return
-        with open(path, 'rb') as f:
-            original = File(f, name=original)
-
+        original_path = os.path.join(self.ROOT, original)
+        try:
+            with open(original_path, 'rb') as f:
+                original = File(f, name=original)
+
+                preview = metadata.get('preview')
+                if preview:
+                    preview_path = os.path.join(self.ROOT, preview)
+                    try:
+                        with open(preview_path, 'rb') as f:
+                            preview = File(f, name=preview)
+                            self.save(metadata, original, instance, preview)
+                    except IOError:
+                        self.report.error("Cannot open path", preview_path)
+                else:
+                    self.save(metadata, original, instance)
+        except IOError:
+            self.report.error("Cannot open path", original_path)
             preview = metadata.get('preview')
             if preview:
-                path = os.path.join(self.ROOT, preview)
-                with open(path, 'rb') as f:
-                    preview = File(f, name=preview)
-                    self.save(metadata, original, instance, preview)
-            else:
-                self.save(metadata, original, instance)
+                preview_path = os.path.join(self.ROOT, preview)
+                try:
+                    with open(preview_path, 'rb') as f:
+                        pass
+                except IOError:
+                    self.report.error("Cannot open path", preview_path)
```

Comment:
If you're checking for the existence of the `preview_path` both in the `try` and in the `except` blocks, then why not just make it its own block?

```python
try:
    with open(original_path, 'rb') as f:
        ...

except OSError as e:
    self.report.error(...)

if preview:
    try:
        with open(preview_path, 'rb') as f:
            ...

    except OSError:
        self.report.error(...)
```

That would avoid some indentation levels, as well as some code duplication.